demisto_sdk/commands/common/native_image.py

```diff
@@ -0,0 +1,144 @@
+import logging
+from typing import Dict, List, Optional
+
+from pydantic import BaseModel
+
+from demisto_sdk.commands.common.handlers import JSON_Handler
+from demisto_sdk.commands.common.tools import extract_docker_image_from_text, get_dict_from_file
+
+json = JSON_Handler()
+logger = logging.getLogger('demisto-sdk')
+
+
+class NativeImage(BaseModel):
+    supported_docker_images: List[str]
+    docker_ref: str
+
+
+class IgnoredContentItem(BaseModel):
+    id: str
+    reason: str
+    ignored_native_images: List[str]
+
+
+class NativeImageConfig(BaseModel):
+    native_images: Dict[str, NativeImage]
+    ignored_content_items: List[IgnoredContentItem]
+
+
+def load_native_image_config(native_image_config_file_path: Optional[str] = None) -> Dict:
+    if not native_image_config_file_path:
+        native_image_config_file_path = 'Tests/docker_native_image_config.json'
```

Comment:
Can we have the value `'docker_native_image_config.json'` only once in the code?
It's also in `TestSuite/docker_native_image_config.py`